app/experimenter/experiments/api/v5/queries.py

```diff
@@ -53,7 +54,19 @@ def resolve_outcomes(root, info):
         return Outcomes.all()
 
     def resolve_targeting_config_slug(root, info):
-        return root._text_choices_to_label_value_list(NimbusExperiment.TargetingConfig)
+        return [
+            NimbusExperimentTargetingConfigSlugChoice(
+                label=member.label,
+                value=choice_name,
+                application_values=NimbusExperiment.TARGETING_CONFIGS[
+                    member.value
+                ].application_choice_names,
+            )
+            for (
+                choice_name,
+                member,
+            ) in NimbusExperiment.TargetingConfig.__members__.items()
```

Comment:
Enums are iterable, you don't need to dig into private attributes.

```py
In [7]: for targeting in NimbusConstants.TargetingConfig:
   ...:     print(targeting.name, targeting.value)
   ...: 
NO_TARGETING 
ALL_ENGLISH all_english
US_ONLY us_only
TARGETING_FIRST_RUN first_run
TARGETING_FIRST_RUN_CHROME_ATTRIBUTION first_run_chrome
TARGETING_FIRST_RUN_WINDOWS_1903_NEWER first_run_win1903
TARGETING_HOMEPAGE_GOOGLE homepage_google_dot_com
TARGETING_URLBAR_FIREFOX_SUGGEST urlbar_firefox_suggest_us_en
```

Reply:
Ah, could have sworn that didn't work when I tried it